### ISSUE TITLE
deprecated attributes in the models from the zoo

### ISSUE THREAD

**Issue by owulveryck** [onnx/models#167]
Hello,

**TL;DR:** is there a plan to remove the deprecated attributes? Shouldn't we ensure that all new uploaded models do not use any deprecated attributes?

**Longer:** In the IR, the `auto_padding` is deprecated in most of the operators.
The problem is that a lot of backends is still using it, and some of the models from the zoo are using this attribute for `maxpool` or `convolution`.

We may have some backends that don't implement this behavior and yet be compatible with ONNX. 
As an example, I am implementing an [ONNX decoder](https://github.com/owulveryck/onnx-go) in Go and I  am using [the Gorgonia library](https://github.com/gorgonia/gorgonia) as a backend.
The problem I encounter is that I have to implement those attributes if I want to actually execute those models, and from times to time it is not trivial. 

- Shouldn't we ensure that all new uploaded models do not use any deprecated attributes?
- What is the plan to manage the lifecycle of the models in the zoo?

**Comment by prasanthpul** [comment]
Depends on the opset you are trying to be compatible with. ONNX Runtime, for example, supports all opsets 7 and higher.
In the model zoo, the models are marked to indicate which opset they are based on.

When new models are uploaded, they should be validated against the opset they claim to be. (Note, we dont have this type of automated checking currently but its on the wish list)

**Comment by prasanthpul** [comment]
If there are deprecated attributes in the definition of an op for a given opset, I agree we should address that. Probably we shouldn't have deprecated attributes because it's confusing as to whether it should be implemented. We need to be more clear about the spec. @onnx/sig-operators should address this

**Comment by maxwillzq** [comment]
Where we can download update-to-date onnx model for benchmark test?
